server.py

```diff
@@ -77,4 +68,24 @@ def http_server():
     server.serve_forever()
 
 
+def swarm_node_commands(request_command):
+    result = -1
+
+    if request_command['command'] == "generate_address":
+        result = generate_address()
+    elif request_command['command'] == "get_tips":
+        result = get_tips(int(request_command['type']))
+    elif request_command['command'] == "send_transfer":
+        if 'debug' not in request_command:
+            debug = 0
+        else:
+            debug = int(request_command['debug'])
```

Comment:
Maybe it could be changed to following code.
Give the `debug`  variable default value instead using if-else clause.
```python
            debug = 0
            if 'debug' in request_command:
                debug = int(request_command['debug'])
```